src/main/java/org/csanchez/jenkins/plugins/kubernetes/volumes/workspace/EmptyDirWorkspaceVolume.java

```diff
@@ -60,16 +66,31 @@ public Boolean getMemory() {
         return memory != null && memory;
     }
 
+    @CheckForNull
+    public String getRequestsSize() {
+        return requestsSize;
+    }
+
+    @DataBoundSetter
+    public void setRequestsSize(@CheckForNull String requestsSize) {
+        this.requestsSize = Util.fixEmptyAndTrim(requestsSize);
+    }
+
     @Override
     public Volume buildVolume(String volumeName, String podName) {
         return new VolumeBuilder()
                 .withName(volumeName)
                 .withNewEmptyDir()
+                .withNewSizeLimit(getRequestsSizeOrDefault())
                 .withMedium(getMedium())
                 .endEmptyDir()
                 .build();
     }
 
+    private String getRequestsSizeOrDefault() {
+        return StringUtils.defaultString(getRequestsSize(), "10Gi");
```

Comment:
For backward compatibility, the default should remain unset. Otherwise users using more than the defined default will be evicted when upgrading to the new plugin version.